snapshotter/Makefile

```diff
@@ -11,12 +11,12 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-all: snapshotter
+all: naive
 
-snapshotter: *.go
-	go build -o snapshotter ./cmd/snapshotter
+naive: *.go
+	go build -o naive_snapshotter ./cmd/naive
```

Comment:
We should update the readme to point to the binary built here instead of snapshotter.

Reply:
Done